tool/site/client/index.html

```diff
@@ -131,70 +131,20 @@ <h2 class="nhsuk-error-summary__title" id="error-summary-title">
-                <path d="M0 0h24v24H0z" fill="none"></path>
-                <path
-                  d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z">
-                </path>
-              </svg>
-              <span class="nhsuk-action-link__text">Create Prescription(s)</span>
-            </a>
-          </div>
-        </div>
-        <div class="nhsuk-grid-row">
-          <div class="nhsuk-grid-column-full">
-            <a style="margin-bottom: 25px;" class="nhsuk-action-link__link" href="{{base_url}}dispense/release">
-              <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 24 24" aria-hidden="true">
-                <path d="M0 0h24v24H0z" fill="none"></path>
-                <path
-                  d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z">
-                </path>
-              </svg>
-              <span class="nhsuk-action-link__text">Release Prescription(s)</span>
-            </a>
-          </div>
-        </div>
-        <div class="nhsuk-grid-row">
-          <div class="nhsuk-grid-column-full">
-            <a style="margin-bottom: 25px;" class="nhsuk-action-link__link" href="{{base_url}}search">
-              <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 24 24" aria-hidden="true">
-                <path d="M0 0h24v24H0z" fill="none"></path>
-                <path
-                  d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z">
-                </path>
-              </svg>
-              <span class="nhsuk-action-link__text">Check Prescription(s) status</span>
-            </a>
-          </div>
-        </div>
-        <div style="margin-bottom: 50px;" class="nhsuk-grid-row">
-          <div class="nhsuk-grid-column-full">
-            <a style="margin-bottom: 25px;" class="nhsuk-action-link__link" href="{{base_url}}validate">
-              <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 24 24" aria-hidden="true">
-                <path d="M0 0h24v24H0z" fill="none"></path>
-                <path
-                  d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z">
-                </path>
-              </svg>
-              <span class="nhsuk-action-link__text">Validate a FHIR Resource</span>
-            </a>
+            <button class="nhsuk-button" type="submit" onclick="updateAuthMethod('cis2')">CIS 2</button>
```

Comment:
why has this been added back? the isLogin stuff should've gone in the previous pr right?

Reply:
removed